extension/js/common/settings.ts

```diff
@@ -369,6 +383,19 @@ export class Settings {
     await Ui.delay(1000);
   }
 
+  /**
+    * determines how to treat old values when changing account
+    */
+  private static getOverwriteMode = (key: string): 'fallback' | 'forget' | 'keep' => {
+    if (key.startsWith('google_token_') || ['uuid', 'rules', 'openid', 'full_name', 'picture', 'sendAs'].includes(key)) { // old value should be used if only a new value is missing
```

Comment:
not sure if we should migrate 'sendAs'

Reply:
We probably don't need to, but it probably also doesn't hurt to - it will get updated next time they open compose window (I think).